discovery-provider/src/tasks/users.py

```diff
@@ -210,6 +211,15 @@ def parse_user_event(
         if metadata_overrides["location"]:
             user_record.location = metadata_overrides["location"]
 
+    # If the multihash is updated, fetch the metadata (if not fetched) and update the associated wallets column
+    if event_type == user_event_types_lookup["update_multihash"] and user_record.handle:
+        user_ipfs_metadata = metadata_overrides
+        if not user_record.is_creator:
```

Comment:
should this be if user_record.is_creator?

larger point though, maybe the is_creator field isn't the right one to check. Maybe we should just check the existence of metadata_multihash?

Here & below

Reply:
The reason I check the is_creator field is because we already fetch the ipfs metadata on line 193 if the user is a creator. This checks that on update to metadata, that if the user's metadata is not fetch from ipfs, we will do it.
If I changed the current function  `get_metadata_overrides_from_ipfs` to fetch the ipfs metadata if the multihash exists, I didn't want the start executing lines 197-212 of this file for users who this didn't run for previously. I figured I would try to isolate the potential for bugs if that metadata isn't right and it overrides it for some reason. 
Am open to changing it though. Thoughts?